specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/secureScore.json

```diff
@@ -0,0 +1,615 @@
+          "type": "integer",
+          "format": "int32"
+        },
+        "notApplicableResources": {
+          "description": "Number of not applicable resources in the control",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "disabledResources": {
+          "description": "Number of not resources that are disabled by policy in the control",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "definition": {
+          "$ref": "#/definitions/SecureScoreControlDefinitionItem"
+        }
+      }
+    },
+    "SecureScoreControlsDefinitionList": {
+      "description": "Page of a secure score controls definition list",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of secure score controls definition in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoreControlDefinitionItem"
+          }
+        },
+        "nextLink": {
+          "description": "The URI to fetch the next page.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SecureScoreControlDefinitionItem": {
+      "description": "Secure Score Control's Definition information",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecureScoreControlDefinitionItemProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecureScoreControlDefinitionItemProperties": {
+      "description": "Secure Score Control Definition Properties",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string"
+        },
+        "description": {
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string",
+          "maxLength": 256
+        },
+        "maxScore": {
+          "description": "Maximum control score (0..10)",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 10,
+          "exclusiveMinimum": false
+        },
+        "source": {
+          "description": "Source object from which the control was created",
+          "readOnly": true,
+          "$ref": "#/definitions/SecureScoreControlDefinitionSource"
+        },
+        "assessmentsDefinition": {
+          "description": "array of assessments metadata ids",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoreControlDefinitionAssessmentsId"
```

Comment:
please add an object to the common that will be an "AzureResourceLink" that will have the same properties